documentation/assemblies/assembly-kafka-connect-external-configuration.adoc

```diff
@@ -11,7 +11,8 @@
 
 = Using external configuration and secrets
 
-Connectors are created, reconfigured, and deleted using the Kafka Connect HTTP REST interface, or by using `KafkaConnectors`. For more information on these methods, see xref:con-creating-managing-connectors-str[]. The connector configuration is passed to Kafka Connect as part of an HTTP request and stored within Kafka itself.
+Connectors are created, reconfigured, and deleted using the Kafka Connect HTTP REST interface, or by using `KafkaConnectors`. For more information on these methods, see link:{BookURLDeploying}#con-creating-managing-connectors-str[Creating and managing connectors^].
```

Comment:
Please can I check my understanding.

When using the `{BookURLDeploying}` attribute or the other `{Book}` attributes, it is necessary to use `link` instead of `xref` and to specify the module name in the square brackets. The caret symbol opens the link in a new tab. Is that right?

Does Asciidoctor complain if this advice is not followed?

We should update the documentation contributors guide with information on linking between different guides in the Strimzi documentation.

Reply:
What do you think about specifying the name of the guide? So, that would be:

See [Creating and managing connectors](#) in the _Deploying Strimzi_ guide.

Reply:
Thanks Daniel. We don't need to open in a new tab, but it probably makes sense if we're navigating to anther guide. For example, in the deployment guide there might be a link to a schema table in the using guide. It's better to have the user guide open in another tab for reference. 

Link is necessary, as we're navigating other paths. xref doesn't work.

Might be an idea to specify the guide name to make it clearer we're moving somewhere else.  :+1:

Reply:
I've added in the _Deploying Strimzi_ and _ Using Strimzi_ guide where appropriate.
In callouts where there might be several links, it didn't look right.